rollup.config.js

```diff
@@ -84,7 +87,18 @@ export default [
       applySwRegistration({
         htmlFileName: 'index.html'
         // base: 'lockdown/'
-      })
+      }),
+      {
+        name: 'minify-css',
+        writeBundle() {
```

Comment:
A more idomatic way in rollup is to use the generateBundle phase, rewrite the assets in memory:

Pseudocode:
```
generateBundle(options, bundle) {
  for (const [name, file] of Object.entries(bundle) {
    if (name.endsWith('.css') {
      file.source = // inify
    }
  }
  }
}
```

Reply:
Nevermind, the copy plugin doesn't place assets in the bundle.